transport/http/outbound.go

```diff
@@ -226,11 +226,17 @@ func (o *Outbound) CallOneway(ctx context.Context, treq *transport.Request) (tra
 		return nil, yarpcerrors.InvalidArgumentErrorf("request for http oneway outbound was nil")
 	}
 
-	_, err := o.call(ctx, treq)
+	// res is used to close the response body to avoid memory/connection leak
+	// even when the response body is empty
+	res, err := o.call(ctx, treq)
```

Comment:
I would advise to enhance the test for the method `CallOneway` - we should have a test very similar to `TestCallSuccess`. We should test:
- Success with response (even if it is callOneway)
- Success with no response and empty payload
- Errors